source/_layouts/default.latte

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <html lang="cs">
-{include "head"}
+{include "_snippets/head.latte"}
```

Comment:
Files need full relative paths, former "head" naming caused conflicts in files with same base name